### PR TITLE
Migrate to python driven chip tests

### PR DESCRIPTION
**Pull request by Apollon77**
This PR is adjusting the former "integrated in chip tool testing" to the new yaml based python runner using chip tool and also a first pure python test case. For this the PR removes the old JS based test runner and just offers comparable test apps and add instructions (and CI) to run the tests using the Matter SDK. For this the full matter sdk checkout is needed.

This PR additionally includes:
* two more commissiong flow adjustments found while finalizing the tests
* some minimal BLE tweaks
* allow the "command line "logic to be called with --name additionally to -name as before.

The latter two changes got committed  unintended ... sorry

### CODE REVIEW

File: packages/matter-node-ble.js/src/ble/NobleBleChannel.ts [hunk 1]
```diff
@@ -79,6 +80,10 @@ export class NobleBleCentralInterface implements NetInterface {
                 `Peripheral ${address.peripheralAddress} is already connected. Only one connection supported right now.`,
             );
         }
+        if (peripheral.state !== "disconnected") {
+            // Try to cleanup strange in between states
```
Comment (lauckhart) [inline packages/matter-node-ble.js/src/ble/NobleBleChannel.ts:84]: Strange = storage?

File: packages/matter-node.js/test/IntegrationTest.ts [hunk 2]
```diff
@@ -963,6 +962,7 @@ describe("Integration Test", () => {
             assert.deepEqual(pushedUpdates, [{ value: true, time: startTime + 2 * 1000 + 100 }]);
         });
 
+        /*
```
Comment (lauckhart) [inline packages/matter-node.js/test/IntegrationTest.ts:965]: Could include a note on why commented and/or when to uncomment

**Review by lauckhart — APPROVED** [review]

File: packages/matter-node.js/test/IntegrationTest.ts (hunk 2 above)
Comment (Apollon77) [reply packages/matter-node.js/test/IntegrationTest.ts:965]: agreed

File: packages/matter-node-ble.js/src/ble/NobleBleChannel.ts (hunk 1 above)
Comment (Apollon77) [reply packages/matter-node-ble.js/src/ble/NobleBleChannel.ts:84]: no really 

strange "in between states"

Comment (Apollon77) [reply packages/matter-node-ble.js/src/ble/NobleBleChannel.ts]: adjustes text

**Pull request merged**